test/dataloaders-test.ts

```diff
@@ -1,46 +1,52 @@
 import assert from "node:assert";
 import {readFile, stat, unlink, utimes} from "node:fs/promises";
 import os from "node:os";
+import {normalizeConfig} from "../src/config.js";
 import {type LoadEffects, Loader} from "../src/dataloader.js";
 
 const noopEffects: LoadEffects = {
   logger: {log() {}, warn() {}, error() {}},
   output: {write() {}}
 };
 
-describe("Loader.find(root, path)", () => {
+describe("Loader.find(root, path)", async () => {
```

Comment:
This is really evil! If you have an `async` callback to `describe`, it silently skips the tests! As a result we are currently running only 615 of 650 tests in main. 😬